docs/src/tutorials/standalone/Canopy/changing_canopy_parameterizations.jl

```diff
@@ -1 +1,172 @@
+# which takes the parameters object directly:
+radiative_transfer_parameters =
+    Canopy.BeerLambertParameters(FT; G_Function, α_PAR_leaf, α_NIR_leaf, Ω);
+radiative_transfer = Canopy.BeerLambertModel(radiative_transfer_parameters);
+
+# All three of these methods will construct a `BeerLambertModel`;
+# the first will use the default parameters, while the second and third use the custom parameters.
+# The method you choose will depend on your use case and whether you want to
+# use the default parameters or provide custom ones.
+# The set of available constructors for all ClimaLand models can be found in the "APIs" section of the documentation.
+
+# Now we can create the `CanopyModel` model with the specified energy and radiative transfer
+# parameterizations passed as [keyword arguments](https://docs.julialang.org/en/v1/manual/functions/#Keyword-Arguments).
+model = Canopy.CanopyModel{FT}(
+    domain,
+    (; atmos, radiation, ground),
+    LAI,
+    earth_param_set;
+    energy,
+    radiative_transfer,
+);
+
+# Define a function to set initial conditions for the prognostic variables.
+# Since these are specific to the model physics, the contents here differ from
+# the soil example, but the function structure remains the same.
+# The variables initialized here are described in the Model Equations section
+# of the documentation.
+# Note that here we previously set the initial condition for the canopy energy model's
+# prognostic temperature. Since we're using the `PrescribedCanopyTempModel`,
+# we do not need to set it here.
+function set_ic!(Y, p, t0, model)
+    ψ_leaf_0 = FT(-2e5 / 9800);
+    (; retention_model, ν, S_s) = model.hydraulics.parameters;
+    S_l_ini = Canopy.PlantHydraulics.inverse_water_retention_curve(
+        retention_model,
+        ψ_leaf_0,
+        ν,
+        S_s,
+    );
+    Y.canopy.hydraulics.ϑ_l.:1 .=
+        Canopy.PlantHydraulics.augmented_liquid_fraction.(ν, S_l_ini);
+end
+
+# Since we'll want to make some plots, let's set up an object to save the
+# model output periodically, as we did for the soil tutorial.
+diag_writer = ClimaDiagnostics.Writers.DictWriter();
+diagnostics = ClimaLand.Diagnostics.default_diagnostics(
+    model,
+    start_date;
+    output_vars = ["ct", "trans"],
+    output_writer = diag_writer,
+    average_period = :hourly,
+);
+
+# Now construct the `LandSimulation` object, which contains the model
+# and additional timestepping information. This is identical to the soil example.
+simulation = LandSimulation(
+    start_date,
+    end_date,
+    dt,
+    model;
+    set_ic!,
+    user_callbacks = (),
+    diagnostics,
+);
+
+# Now we can run the simulation!
+solve!(simulation);
+
+# Let's plot some results, for example diurnally averaged canopy temperature and transpiration over time:
+LandSimVis.make_diurnal_timeseries(
+    simulation;
+    short_names = ["ct", "trans"],
+    plot_stem_name = "default_canopy",
+);
+# ![](ct_canopy_parameterizations.png)
```

Comment:
These plots dont show up, I think you need to change the stem_name to "canopy_parameterizations"